spec/requests/disk_controller_spec.rb

```diff
@@ -0,0 +1,125 @@
+# frozen_string_literal: true
+require "rails_helper"
+
+RSpec.describe "FileBrowser/Disk" do
+  let(:user) { nil }
+  before do
+    sign_in(user) if user
+  end
+  describe "#index" do
+    context "when not logged in" do
+      it "returns forbidden" do
+        get "/file_browser/disk.json"
+
+        expect(response).to be_forbidden
+      end
+    end
+    context "when logged in as staff" do
```

Comment:
should we have a test for admin as well?

Reply:
I didn't because admins have global permissions to everything - if a staff member can do it then an admin definitely can.